index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Microverse Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <div>
+                <a  class="logo" href="#">My Logo</a>
+            </div>
+            <div class="menu-placeholder">
+                <a href="#"><img src="./images/Union.svg" alt="menu icon"></a>
+            </div>
```

Comment:
- [ ] Please change the `<div>` tags to `<nav>`, this is so as to enhance the semantics of your HTML code.

Reply:
Yes, makes sense since it will be a navigation menu. Thank you.